polaris/ocean/tests/single_column/single_column.cfg

```diff
@@ -0,0 +1,66 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 100
+
+# Depth of the bottom of the ocean
+bottom_depth = 400.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
+
+# config options for single column testcases
+[single_column]
+
+# the size of the domain in km in the x dimension
+# we want to use the minimum number of cells so we set the dimensions to be
+# less than or equal to the minimum expected resolution
+lx = 1.0
+
+# the size of the domain in km in the x dimension
+# we want to use the minimum number of cells so we set the dimensions to be
+# less than or equal to the minimum expected resolution
+ly = 1.0
+
+# Surface temperature
+surface_temperature = 20.0
+
+# Temperature gradient in the mixed layer in degC/m
+temperature_gradient_mixed_layer = 0.0
+
+# The temperature below the mixed layer
+temperature_difference_across_mixed_layer = 0.0
+
+# Temperature gradient below the mixed layer
+temperature_gradient_interior = 0.01
+
+# Depth of the temperature mixed layer
+mixed_layer_depth_temperature =  25.0
+
+# Surface salinity
+surface_salinity = 35.0
+
+# Salinity gradient in the mixed layer in PSU/m
+salinity_gradient_mixed_layer = 0.0
+
+# The salinity below the mixed layer
+salinity_difference_across_mixed_layer = 1.0
+
+# Salinity gradient below the mixed layer
+salinity_gradient_interior = 0.0
+
+# Depth of the salinity mixed layer
+mixed_layer_depth_salinity = 0.0
+
+# coriolis parameter
+coriolis_parameter = 1.0e-4
```

Comment:
I would put these in a `[cvmix]` or `[single_column_cvmix]` section in a `cvmix.cfg` in the `cvmix` directory.  I don't think you want to assume that all single column tests will use these config options.

Reply:
@xylar I figured that when another test case was added they could decide how many of these options to use and how many to branch into different sections (I'm guessing most of the test cases will be testing different vertical mixing schemes and it won't matter exactly what the initial profile is). I'm fine moving these to a `cvmix` section, but I think then we would want to move the initial state set up as well to the `cvmix` directory since that invokes cfg options in that section. And then also move `forward.yaml` to the `cvmix` directory because it contains cvmix-related options?

Reply:
Okay, that's fine.  I guess I thought from the description that "single column" was a really generic name and could encompass a super wide variety of tests.  But I'm fine if keeping things as they are if that's not the direction things are looking like they'll go.